### ISSUE TITLE
Support GS1-128 barcodes

### ISSUE THREAD

**Issue by Anisi** [milon/barcode#107]
Hi
Does this library support GS1-128 bar-codes?

**Comment by nejcdev** [comment]
idk

**Comment by propeller-aaron** [comment]
echo DNS1D::getBarcodeHTML('4445645656', 'GS1-128');

I am having some trouble with it though.

**Comment by propeller-aaron** [comment]
> echo DNS1D::getBarcodeHTML('4445645656', 'GS1-128');
> 
> I am having some trouble with it though.

I figured it out, Composer was not pulling in the Minor version of the repo.

**Comment by milon** [comment]
Yes — use barcode type **\`GS1-128\`** (or \`C128\` with an appropriate GS1 payload).

```php
DNS1D::getBarcodePNG(\$gs1Payload, 'GS1-128', 2, 60);
```

Closing as answered.
